controllers/single_page/dashboard/system/environment/graphql.php

```diff
@@ -90,10 +90,11 @@ public function update_entity_settings()
                         $config->save('concrete5_graphql_websocket::websocket.debug', false);
                         $servers = (array) ($config->get('concrete5_graphql_websocket::websocket.servers'));
                         $config->save('concrete5_graphql_websocket::websocket.servers', []);
+                        $websocketService = $this->app->make(WebsocketService::class);
                         foreach ($servers as $port => $pid) {
```

Comment:
foreach ($servers as $pid) {

Reply:
I didn't touch this line ([was already like that](https://github.com/lemonbrain-mk/graphql_websocket/blob/1.2.5/controllers/single_page/dashboard/system/environment/graphql.php#L93)), but I agree: getting the `$port` is useless here. :+1: